Sort.py

```diff
@@ -0,0 +1,45 @@
+def bed_sort(bed_file):
```

Comment:
Supposed that we get list of bed entries here as parameter. As described here https://github.com/AleksandrSl/pybedtools/issues/4